core/src/main/java/feast/core/model/FeatureSet.java

```diff
@@ -359,9 +346,4 @@ public boolean equals(Object obj) {
     }
     return this.equalTo(((FeatureSet) obj));
```

Comment:
A random individual comment here, because I was just now looking at the labels PR when pulled in for our internal backport, and noticing that we did not update `FeatureSet`'s equality for labels.

I don't have this branch checked out at the moment, but I believe `equalTo` is effectively unused now because it's only usage was in `SpecService` for determining if version should increment—`equalTo` can probably be folded into the `equals` method now.

We can send a quick patch for including labels in `equals`, unless you want to do it here if you go ahead and consolidate the two methods.

Reply:
I think i'd love for it to be a patch just so this PR is a little less overwhelming to digest, haha.

Reply:
Patch is here: #690 
I'll consolidate the equality methods in this PR after that PR is merged in

Reply:
This is done